src/Items/ItemHandler.cpp

```diff
@@ -531,12 +531,11 @@ void cItemHandler::OnFoodEaten(cWorld * a_World, cPlayer * a_Player, cItem * a_I
 
 short cItemHandler::GetDurabilityLossByAction(eDurabilityLostAction a_Action)
 {
-	switch ((int)a_Action)
+	switch (a_Action)
 	{
 		case dlaAttackEntity: return 2;
 		case dlaBreakBlock:   return 1;
 	}
-	return 0;
```

Comment:
This line needs to stay for non-clang compilers. At least MSVC generates a warning without it. Wrap it in `#ifndef __clang__`